src/NLog/LayoutRenderers/ExceptionLayoutRenderer.cs

```diff
@@ -409,7 +409,7 @@ protected virtual void AppendSerializeObject(StringBuilder sb, Exception ex)
         private static List<ExceptionRenderingFormat> CompileFormat(string formatSpecifier)
         {
             List<ExceptionRenderingFormat> formats = new List<ExceptionRenderingFormat>();
-            string[] parts = formatSpecifier.Replace(" ", string.Empty).Split(new string[] { "," }, StringSplitOptions.RemoveEmptyEntries);
+            string[] parts = formatSpecifier.SplitAndTrimTokens(',');
```

Comment:
we've lost remove space here, not sure if this is an issue.

Reply:
Yes it would support a format like `${exception:format=to string}`. But I think that is a buggy sideeffect of trying to support  `${exception:format=tostring }` (With trailing space).

Reply:
I think also, thanks for checking